manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -120,6 +120,11 @@ spec:
               # network namespaces in `/proc` to obtain descriptors for entering pod network
               # namespaces. There does not appear to be a more granular capability for this.
               - SYS_ADMIN
+              # While we run as a 'root' (UID/GID 0), since we drop all capabilities we lose
+              # the typical ability to read/write to folders owned by others.
+              # This can cause problems if the hostPath mounts we use, which we require write access into,
+              # are owned by non-root. DAC_OVERRIDE bypasses these and gives us write access into any folder.
+              - DAC_OVERRIDE
```

Comment:
Did we try `CAP_FOWNER` and find it wanting (I'm not 100% it would solve this but from a cursory read it looks like it's specifically targeted at ownership mismatches)?

That won't help if something is missing the right RWX flags regardless of ownership, but stuff we create _should_ have those bits set correctly.

Reply:
I didn't try it but I don't think it will work. The folder is `drwxr-xr-x` and owned by `random-user-that-is-not-root`. So we don't have write access